.github/workflows/app_build.yml

```diff
@@ -292,7 +292,7 @@ jobs:
       uses: actions/cache@v2
       with:
         path: build/
-        key: ${{ matrix.os }}-build-cache-${{ secrets.CACHE_KEY }}
+        key: ${{ matrix.os }}-build-cache-v1-${{ secrets.CACHE_KEY }}
```

Comment:
I think you can just change the cache key secret right?

Reply:
Yeah I forgot  about that, thanks for the reminder